jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue

```diff
@@ -37,17 +37,15 @@
       </v-row>
   
       <v-row no-gutters>
-        <v-col cols=6><U><B>Key</B></U></v-col>
-        <v-col cols=6><U>Value</U></v-col>
-        <v-col v-if="has_comments" cols=6 class="text--secondary"><U>Comment</U></v-col>
+        <v-col cols=12><U><B>Key | Value </B></U></v-col>
+        <v-col v-if="has_comments" cols=12 class="text--secondary"><U>Comment</U></v-col>

```

Comment:
Do we need the header at all or is it self-explanatory?  If we do keep it, the value isn't bolded in the contents, so let's not have it bolded in the header either.
```suggestion
        <v-col cols=12><U><B>Key</B></U> | <U>Value</U></v-col>
        <v-col v-if="has_comments" cols=12 class="text--secondary"><U>Comment</U></v-col>

```

Reply:
I didn't like the header because I don't think it adds much value. I was trying to keep it the same as before, but I don't think we should keep them.

Reply:
ok, maybe check with @Jenneh on the final styling and if she agrees this can be removed entirely?

Reply:
@kecnry,
@Jenneh  also suggested that we remove the headers.